src/commands/realSwipe.ts

```diff
@@ -117,6 +124,10 @@ export async function realSwipe(
     touchPoints: [startPosition],
   });
 
+  if (options.touchMoveDelay) {
+    await wait(options.touchMoveDelay);
```

Comment:
I know that I made a similar thing in the press but is it possible to use cy.wait here instead? If no, can you please reuse the same wait function between 2 commands

Reply:
Just saw you had mentioned "press" command, so what I've found should be what you meant. I can extract this into a tiny util file, and reuse in both places.

Reply:
Yeah that would be perfect. Just left a comment why it is there

Reply:
Done